docs/networking.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Networking
+subtitle: Connecting ev3dev to the internet and other devices
+---
+
+If you want to access your ev3dev installation from a PC or install new software
+on your EV3, you'll need to connect to a network. Here are some resources to
+help you out.
+
+# Connecting to the internet
+
+You have a few options for connecting to the internet.
+
+### With a Wi-Fi dongle
+
+The simplest route is to purchase a Wi-Fi dongle and plug it into the EV3's USB
```

Comment:
_If you have a Wi-Fi dongle_ (e.g. the LEGO-compatible one), I think Wi-Fi is the simplest. If you don't, USB will be the easiest as you mentioned. The reason that I put Wi-Fi first was because Windows has issues with USB sharing (you frequently have to disable and then re-enable the option).

How does this sound: I reword and restructure this so that it says something to that effect. If you have a Wi-Fi dongle, use it; otherwise use USB.